### PR TITLE
add option to change default directory

### PR DESCRIPTION
**Pull request by teeann**

### CODE REVIEW

**Comment by shin2903** [comment]
All checks have failed because Python 3.5 doesn't support f-string.

File: bin/tanner [hunk 1]
```diff
@@ -17,7 +17,9 @@ def main():
     """)
     parser = argparse.ArgumentParser()
     parser.add_argument("--config", help="tanner config")
+    parser.add_argument("--base-dir", help="tanner base directory", default='/opt/tanner')
```
Comment (rnehra01) [inline bin/tanner]: Newline

**Comment by rnehra01** [comment]
@afeena Should we create a default dir that doesn't require sudo privileges?

File: bin/tanner (hunk 1 above)
Comment (rnehra01) [reply bin/tanner]: Add an empty line after this just like we had before.

**Comment by rnehra01** [comment]
@afeena You are talking about [this](https://github.com/mushorg/tanner/blob/master/setup.py#L13), I guess. I'm not sure how to to do that. @shin2903 maybe can you take a look?

File: bin/tanner (hunk 1 above)
Comment (shin2903) [reply bin/tanner]: done :)

**Comment by afeena** [comment]
For now I suggest to use `.local/share/tanner` or `.tanner` in home directory since we use it only for storing `data` by default :)

**Comment by glaslos** [comment]
@gaochua did you see the suggestion on the default directory?

**Comment by gaochua** [comment]
@glaslos Yes I am working to change the default directory in `setup.py`. This may take a while, as I'm quite busy at the moment.

**Comment by afeena** [comment]
please open new PR if needed

**Pull request closed without merging**